ci-operator/jobs/openshift/cluster-api-provider-aws/openshift-cluster-api-provider-aws-master-presubmits.yaml

```diff
@@ -66,6 +66,57 @@ presubmits:
           name: prow-job-cluster-launch-installer-e2e
         name: job-definition
     trigger: ((?m)^/test( all| e2e-aws),?(\s+|$))
+  - agent: kubernetes
+    always_run: false
+    optional: true
+    branches:
+    - master
+    context: ci/prow/e2e-aws-operator
+    decorate: true
+    name: pull-ci-openshift-machine-api-operator-master-e2e-aws-operator
```

Comment:
`machine-api-operator` in the name? maybe cluster-api-provider-aws?

Reply:
Fixed